impress-org/givewp-next-gen
# Security: no longer store payment intent secret

## Description

This is related to: https://github.com/impress-org/givewp/pull/6836

As mentioned in that PR, we're not longer storing the intent secret as it's unnecessary and unwise.

## Affects

Shouldn't affect anything, just a removal.

## Testing Instructions

1. Make a Stripe donation
2. Check that the `_give_stripe_payment_intent_client_secret` donation meta is not stored

## Pre-review Checklist
